jQuery/honeyHeadquarters.js

```diff
@@ -1,4 +1,27 @@
 // $.ajax('https://sheetsu.com/apis/v1.0/64b5c3f8').done(function(data) {
-$.ajax('http://localhost:3000/citizens').done(function(data) {
-  console.log(data);
+$.ajax('http://localhost:3000/citizens').done(function(citizens) {
+  function getUniqueValuesForField(field, citizens) {
+    const values = ['All'];
+    citizens.forEach((c) => {
```

Comment:
You're using a fat arrow syntax here. This is es6. ya can't do it if you want this to work in more than just chrome/firefox. `function() { ... } ` won't kill you 💯